### PR TITLE
Fixed setup error

### PR DESCRIPTION
**Pull request by Noviny**
Fixed error in type.less that caused error in LESS compiler, but had to
remove responsive headings.

Also moved from React.render to reactDOM.render, and a few other small
fixes.

### CODE REVIEW

File: client/scripts/home.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 var React = require('react');
+var reactDOM = require('react-dom');
```
Comment (JedWatson) [inline client/scripts/home.js]: This should be proper cased like React for consistency (i.e. `ReactDOM`)

File: keystone.js [hunk 2]
```diff
@@ -22,6 +22,8 @@ keystone.init({
 	'brand': 'SydJS',
 	'back': '/me',
 
+	'db': 'sydjs-site',
```
Comment (JedWatson) [inline keystone.js]: is this something that was breaking for you? that should be the default database used, it's generated from the project name

**Comment by Noviny** [comment]
Thought the db was required but removing it did not cause it to break, so it has been taken back out.

Should now be ready to pull.

This addresses issue #86

**Comment by Noviny** [comment]
@JedWatson

**Comment by JedWatson** [comment]
Looks great - thanks @Noviny!

**Pull request merged**